libs/components/native/Screens/Login/index.tsx

```diff
@@ -0,0 +1,80 @@
+// Import
```

Comment:
Does the wobble still trigger is a user clicks the button twice in a row without a change in the user input?

Reply:
no the user has to change input :/

Reply:
wasnt motivated enough to sort out - can have another look if u want